app/dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16-slim
+
+WORKDIR /usr/src/app
+
+COPY ./package.json ./
+COPY ./yarn.lock ./
```

Comment:
This line can be taken away as it will be regenerated with the line below ensuring that it is congruent with the package.json of the container